### DISCUSSION TITLE
Cost basis (purchase_price) for assembled stock

### DISCUSSION THREAD

**Discussion by tomd-tc** [inventree/InvenTree#8637] [General]
Good evening,
Again - I appreciate the InvenTree product and its flexibilities but I do have another question:

InvenTree manages all components we need to manufacture/build a PC (ingested into the system as part of POs we send to suppliers). We register parts based on serial numbers (trackable - and if available) or just as quantities (eg. boxes, manuals, cords, etc.)

In the stock_stockitem table I see individual stock items we used in a build, with serial numbers (if applicable) and an associated purchase_price.

When I create a build based on BOM, whereby I assign stock based on serial_numbers (trackable parts) and non-trackable parts (eg. a box), I get the rough sense how much a given Unit costs me to build (i.e. the sum of purchase_price and quantity I assume). When I complete the build, and assign a serial number to the consolidated final product (i.e. a final PC in a box), I see the serial_number appear in the database table stock_stockitem, but the purchase_price is NULL.

Is there a way to force Inventree to record the cost to produce (i.e. the sum of the  material's value) to the given output?

Thank you again!

**Comment by matmair** [comment]
This sounds like something https://github.com/inventree/InvenTree/issues/8607 would adress
